src/main/java/frc/robot/subsystems/gripper/Gripper.java

```diff
@@ -94,4 +90,11 @@ public Command getCloseCommand(){
     return new StartEndCommand(this::close, ()-> setPower(0) , this).until(this::getLimSwitchClose);
   }
 
+  @Override
+  public void initSendable(SendableBuilder builder){
+    builder.addBooleanProperty("limSwitch close", this::getLimSwitchClose, null);
+    builder.addBooleanProperty("limSwitch open", this::getLimSwitchOpen, null);
```

Comment:
Please change the text from "limSwitch X" to "Limit Switch X"
Also, rename the functions from getLimSwitchX to isLimitSwitchX (X is Close/Open)